pcsx2/VMManager.cpp

```diff
@@ -746,23 +746,26 @@ void VMManager::UpdateRunningGame(bool resetting, bool game_starting)
 	Console.WriteLn(Color_StrongGreen, fmt::format("  Serial: {}", s_game_serial));
 	Console.WriteLn(Color_StrongGreen, fmt::format("  CRC: {:08X}", s_game_crc));
 
-	UpdateGameSettingsLayer();
-	ApplySettings();
+	if (!swapping_disc)
+	{
+		UpdateGameSettingsLayer();
```

Comment:
Are you sure about this? If the disc being swapped to has a different serial, we'd probably want to apply the latter's gamesettings/ini.

Reply:
I'd already fixed that but forgot to push it lol